### PR TITLE
Add video to the hero

### PR DESCRIPTION
**Pull request by jaredcunha-usds**
DO NOT MERGE THIS

**Why**

Placeholder for video once it is approved and posted YouTube

**What Changed**

Added video and updated button in hero section

Screen Shot 2019-11-01 at 2 40 23 PM

**Choices Made**

Replaced the documentation button in the hero section

**Tickets closed**:

**Future Work**

### CODE REVIEW

File: dpc-web/app/views/public/home.html.erb [hunk 1]
```diff
@@ -223,4 +223,17 @@ http://hl7.org/fhir/us/bulkdata/2019May/index.html">Bulk FHIR specification</a>.
...
+</main>
+
+<div class="ds-c-dialog-wrap" aria-labelledby="dialog-title" aria-hidden="true" role="dialog" id="filter-modal">
+  <div class="ds-c-dialog ds-c-dialog--full ds-u-fill--base" role="document">
+    <header class="ds-c-dialog__header" role="banner">
+      <button class="ds-c-button ds-c-button--transparent ds-c-dialog__close ds-u-color--white" aria-label="Close modal dialog">Close</button>
+    </header>
+
+
+      <style>.codegena{position:relative;width:100%;height:0;padding-bottom:56.27198%;}.codegena iframe{position:absolute;top:0;left:0;width:100%;height:100%;}</style><div class="codegena"><iframe width='500' height='294' src="https://www.youtube.com/embed/aGe5rEDv3g8?&theme=dark&autohide=2&modestbranding=1&rel=0&iv_load_policy=3"frameborder="0"></iframe></div>
```
Comment (jaredcunha-usds) [inline dpc-web/app/views/public/home.html.erb]: Once the DPC video is placed on YouTube, update the URL on this line

**Comment by jaredcunha-usds** [comment]
Screen Shot 2019-11-26 at 10 22 50 AM

The video will have a USDS channel icon in the top right. @amgleason says that is okay since this will be only be temporary until we can host the video on CMS's youtube channel.

File: dpc-web/app/views/public/_hero.html.erb [hunk 2]
```diff
@@ -18,11 +19,14 @@
           Request access</a>
           <% end %>
 
-          <%= link_to docs_path, :class => "ds-c-button ds-c-button--inverse ds-c-button--inverse--alt ds-c-button--big" do %>
```
Comment (ronaldheft-usds) [inline dpc-web/app/views/public/_hero.html.erb]: Does this remove the link to the documentation?

Comment (switzersc-usds) [reply dpc-web/app/views/public/_hero.html.erb]: It changes the docs link in the hero to a link to open the video. There is still a link to the documentation in the header.

Comment (ronaldheft-usds) [reply dpc-web/app/views/public/_hero.html.erb]: Awesome! I assume we're good to merge now @jaredcunha-usds @amgleason?

Comment (jaredcunha-usds) [reply dpc-web/app/views/public/_hero.html.erb]: I'm good if @amgleason is good.

**Pull request merged**